### PR TITLE
Update MIEngine reference and add step filtering to launch schema

### PR DESCRIPTION
**Pull request by chuckries**

### CODE REVIEW

**Comment by chuckries** [comment]
@wesrupert @gregg-miskelly @caslan

File: src/coreclr-debug/install.ts [hunk 1]
```diff
@@ -163,7 +163,7 @@ export class DebugInstaller
             },
             dependencies: {
                 "Microsoft.VisualStudio.clrdbg": "14.0.25430-preview-3098923",
-                "Microsoft.VisualStudio.clrdbg.MIEngine": "14.0.30629-preview-1",
+                "Microsoft.VisualStudio.clrdbg.MIEngine": "14.0.30705-preview-1",
                 "Microsoft.VisualStudio.OpenDebugAD7": "1.0.20628-preview-1",
```
Comment (gregg-miskelly) [inline src/coreclr-debug/install.ts]: You need to update OpenDebugAD7 for step filtering to work (commit f8714e16c880dac148801ad3bf44f12c879d5ace isn't in release-csharp).

**Comment by gregg-miskelly** [comment]
Otherwise LGTM

File: src/coreclr-debug/install.ts (hunk 1 above)
Comment (chuckries) [inline src/coreclr-debug/install.ts]: good catch, thanks.

**Pull request merged**